test/e2e_env/universal/grpc/grpc.go

```diff
@@ -62,7 +62,7 @@ func GRPC() {
 		}, "30s", "1s").Should(Succeed())
 	})
 
-	It("MeshHTTPRoute DOES NOT split the traffic between two gRPC services", func() {
+	It("MeshHTTPRoute does split the traffic between two gRPC services", func() {
```

Comment:
why is this now the case? why is default policy creation tied to MeshHTTPRoute splitting gRPC traffic?

Reply:
To fully switch to targetRef policies we need grpc support and because of the lack of MeshGRPCRoute we can currently support them with MeshHTTPRoute